.travis.yml

```diff
@@ -7,29 +7,40 @@ language: c
 
 compiler:
   - gcc
+  - clang
+
+addons:
+  apt:
+    packages:
+    - axel
+    - cpanminus
+    - libluajit-5.1-dev
+
+cache:
+  apt: true
+  directories:
+  - download-cache
 
 env:
   global:
     - NGINX_VERSION=1.9.15
     - JOBS=3
     - TEST_NGINX_SLEEP=0.006
+    - LUA_DEV=libluajit-5.1-dev LUA_INCLUDE_DIR=/usr/include/luajit-2.0 LUALIB=-lluajit-5.1
   matrix:
-    - LUA=libluajit-5.1-dev LUA_DEV=libluajit-5.1-dev LUA_INCLUDE_DIR=/usr/include/luajit-2.0 LUALIB=-lluajit-5.1
+    - NGINX_VERSION=1.9.15
+    - NGINX_VERSION=1.11.2
 
 install:
-    - sudo apt-get update
-    - sudo apt-get install -qq -y $LUA cpanminus
     - sudo cpanm --notest Test::Nginx > build.log 2>&1 || (cat build.log && exit 1)
-    - wget http://nginx.org/download/nginx-${NGINX_VERSION}.tar.gz
```

Comment:
wget did make sense when nginx was built using ./configure, as for ngx-build, it is not required

as for caching, we cannot cache file, but directory
I am not very familiar with axel, but it would make sense to tell him "use download-cache directory"